flask_api/app/routes.py

```diff
@@ -0,0 +1,176 @@
+import io
+
+from flask import request, jsonify, Blueprint
+from flask_login import current_user, login_user, logout_user
+
+from config import (
+    USERNAME_KEY,
+    PASSWORD_KEY,
+    COORDS_KEY,
+    ROWS_KEY,
+    TREES_KEY,
+    DEFAULT_ROWS_DISTANCE,
+    DEFAULT_TREES_DISTANCE,
+)
+from . import db
+from .helpers import utils
+from .helpers.generate_trees import TreesGenerator
+from .models import User, Orchard
+
+api = Blueprint("api", __name__)
+
+
+@api.route("/healthchecker", methods=["GET"])
+def healthchecker():
+    return jsonify({"status": "I am alive!"}), 200
+
+
+@api.route("/users", methods=["POST"])
+def create_user():
+    name = request.form.get(USERNAME_KEY)
+    if not name:
+        return jsonify({"status": "Name can't be empty."}), 400
+
+    password = request.form.get(PASSWORD_KEY)
```

Comment:
Why are you using `request.form` instead of JSON? What is the content type of requests here?